locale/hu.json

```diff
@@ -31,256 +31,256 @@
     },
     "nui_menu": {
         "misc": {
-            "help_message": "txAdmin Menu enabled, type /tx to open it.\nYou can also configure a keybind at [Game Settings > Key Bindings > FiveM > Menu: Open Main Page].",
-            "menu_not_allowed": "Your identifiers do not match any admin registered on txAdmin.\nIf you are registered on txAdmin, go to Admin Manager and make sure your identifiers are saved.",
-            "action_unauthorized": "You do not have this permission.",
-            "unknown_error": "An unknown error occurred.",
-            "not_enabled": "The txAdmin Menu is not enabled! You can enable it in the txAdmin settings page.",
-            "announcement_title": "Server Announcement by %{author}:",
-            "general_no_perms": "You do not have permissions for this!",
-            "dialog_empty_input": "You cannot have an empty input."
+            "help_message": "A txAdmin rendelkezik játékon bellüli menüvel!\\\\nGyorsgomb [Game Settings > Key Bindings > FiveM > Open the txAdmin Menu] Próbáld ki!",
+            "menu_not_allowed": "Az azonosítód nem egyezik a txAdminban található admin azonosítókkal.\nHa regisztrálva vagy txAdminban, bizonyosodj meg róla hogy el van mentve az Admin Managerben.",
+            "action_unauthorized": "Ehhez nincs engedélyed.",
+            "unknown_error": "Váratlan hiba lépett fel.",
+            "not_enabled": "A txAdmin menü nincs engedélyezve! Engedélyezheted a txAdmin beállítások fülön.",
+            "announcement_title": "Szerver felhívás %{author}:",
+            "general_no_perms": "Ehhez nincs engedélyed!",
+            "dialog_empty_input": "Nem lehet üresen hagyni."
         },
         "frozen": {
-            "froze_player": "You have frozen the player!",
-            "unfroze_player": "You have unfrozen the player!",
-            "was_frozen": "You have been frozen by a server admin!"
+            "froze_player": "Lefagyasztottad a játékost!",
+            "unfroze_player": "Feloldottad a fagyasztást a játékoson!",
+            "was_frozen": "Lefagyasztott egy szerveradmin!"
         },
         "common": {
-            "cancel": "Cancel",
-            "submit": "Submit",
-            "error": "An error occurred",
-            "copied": "Copied to clipboard."
+            "cancel": "Vissza",
+            "submit": "Küldés",
+            "error": "Hiba lépett fel",
+            "copied": "Vágólapra másolva."
         },
         "page_main": {
             "tooltips": {
-                "tooltip_1": "Use %{key} to switch pages & the arrow keys to navigate menu items",
-                "tooltip_2": "Certain menu items have sub options which can be selected using the left & right arrow keys"
+                "tooltip_1": "%{key} a lapozáshoz és [NYILAK] a menüben való navigáláshoz",
+                "tooltip_2": "Néhány menü rendelkezik al opcióval  mely a bal és jobb nyíllal választható"
```

Comment:
`Néhány menü rendelkezik al-opcióval, melyek a bal és jobb nyilakkal választhatóak` would be more correct imo.